pkgs/applications/networking/mailreaders/aerc/runtime-sharedir.patch

```diff
@@ -26,31 +26,32 @@ index d1c755d..1185a96 100644
  DOCS := \
  	aerc.1 \
 diff --git a/config/config.go b/config/config.go
-index 32d07fc..8ffd3e8 100644
+index 3ae26c1..34b596e 100644
 --- a/config/config.go
 +++ b/config/config.go
-@@ -355,6 +355,11 @@ func LoadConfigFromFile(root *string, sharedir string) (*AercConfig, error) {
+@@ -467,6 +467,11 @@ func LoadConfigFromFile(root *string, sharedir string) (*AercConfig, error) {
  			return nil, err
  		}
  	}
 +	if sec, err := file.GetSection("templates"); err == nil {
-+		if key, err := sec.GetKey("template-dirs"); err == nil {
```

Comment:
Any reason why you changed these 2 characters from spaces to tabs?

Reply:
No reason - I think I must have had my editor misconfigured when I was remaking the patch.